### ISSUE TITLE
Feature: Add PostHog feedback widget to the paltform

### ISSUE THREAD

**Issue by isabensusan** [open-sauced/app#1746]
### Type of feature

🍕 Feature

### Current behavior

N/A

### Suggested solution

For the Product Hunt launch, I'd love to capture feedback from the increased traffic of users that we are expecting.

PostHog has a "Survey" widget that we could use to gather this feedback: [(link to docs)](https://posthog.com/tutorials/feedback-interviews-site-apps)

<img width="445" alt="image" src="https://github.com/open-sauced/app/assets/27322879/e2c259cd-cde6-4ef9-971d-ed9c0d148757">

We want to make sure we are not too annoying about it, but pushy enough to get some answers. To achieve this we should only show one survey per user if we implement more than one, and leverage cookies to make sure we don't show the same survey to a user that has already responded.

### Possible Questions (WIP)
My main goal would be to get information about what users actually want to see on their dashboards:

In Insight Pages:
- What type of insights would you like to see about repositories?

In List Pages:
- What type of insights would you like to see about contributors?

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

**Comment by isabensusan** [comment]
@bdougie would love to know your thoughts on this

**Comment by bdougie** [comment]
I think this is a good idea and a better option until we can build our chatbot into the main product. 

## sitegpt stats

I added SiteGPT when the only thing you could do was look at your profile. The model was meant to be trained on our docs and blog posts, but the features are pretty limited and we would be better off adding that to our chrome extension. 

## integrating posthog feedback.

Do you have a link for how to integrate this? Perhaps we can have @OgDev-01 replace the current chat interaction with this.

**Comment by isabensusan** [comment]
yep -- my bad here's the [link to the PostHog docs](https://posthog.com/tutorials/feedback-interviews-site-apps)

**Comment by bdougie** [comment]
For now, let's hide the SiteGPT and this. We have cohorts built in the posthog data. Maybe we can create a cohort for those who those who interacted with insights and lists feature separately. We can present the question to anyone either on the page or who has created an insight page.  

But for now, we can focus on an existing cohort until the cohorts are created.

**Comment by OgDev-01** [comment]
pasting for reference https://posthog.com/tutorials/react-surveys

**Comment by isabensusan** [comment]
@OgDev-01 as we discussed, here's some more context:

For the link you've posted, what we want to implement is [Option 2:  Use PostHog's prebuilt survey UI](https://posthog.com/tutorials/react-surveys#option-2-use-posthogs-prebuilt-survey-ui), that way I can create surveys on the posthog side.

I've tried creating a PostHog survey to see if it shows up, but it doesn't. I've done some debugging and I think the issue might be that we are working with an older version of posthog (the platform itself did alert me that if things weren't working properly that could be the issue)

**What I've tried so far**
- I've created a survey on PostHog, I've enabled surveys through settings and created a survey without targeting any users/urls so that the survey would show up anywhere on the page
- I've tried creating a targeted survey to show up on different specific urls to see if there's an issue with certain pages, but I still couldn't get it t show up
- I've tried creating different types of surveys with no luck


**So my conclussion is that either:**
A. the older version might be the culprit and we need to update it
B. our stack isn't compatible with PostHog prebuilt surveys and we have to create a custom one using their API
